capstone/src/main/java/com/google/sps/servlets/Club.java

```diff
@@ -0,0 +1,28 @@
+package com.google.sps.servlets;
+
+import java.util.List;
+
+/* Contains all information relevant to a club. */
+public class Club {
```

Comment:
BTW, you'll likely have to add getters and setters to access and modify your individual field variables since they're private (and they should be). But you can add them later when you need them.